docker/entrypoint.sh

```diff
@@ -5,16 +5,16 @@ set -eu -o pipefail
 : "${BIND_ON_IP:=$(getent hosts $(hostname) | awk '{print $1;}')}"
 export BIND_ON_IP
 
-# check TEMPORAL_CLI_ADDRESS is not empty
-if [[ -z "${TEMPORAL_CLI_ADDRESS:-}" ]]; then
-    echo "TEMPORAL_CLI_ADDRESS is not set, setting it to ${BIND_ON_IP}:7233"
+# check TEMPORAL_ADDRESS is not empty
```

Comment:
can we set both since `tctl` may still use the longer names?

Reply:
sg